blog/2015-05-21-port-monitoring.adoc

```diff
@@ -0,0 +1,53 @@
+= Port Monitoring of Application Servers with JBoss Tools
+:page-layout: blog
+:page-author: robstryker
+:page-tags: [jbosstools, devstudio, server, wildfly]
+
+You may have noticed over the years, that your server view's context menu has an option labeled "Port Monitoring". You  may have also noticed that when using JBoss servers, nothing was ever available there. Recent commits have finally remedied this decade-long omission of a useful feature. 
+
+== TCP/IP Monitor
+
+The WTP platform includes a TCP/IP monitor as a standalone entity. You can use it manually, which I'll explain now, but the real magic happens when integrated with our server adapters. 
+
+There are two primary pieces to the TCP/IP Monitor. First, there's a preference page, where you can add port mappings and enable or disable monitoring on that port. Secondly, there's a TCP/IP Monitor view. 
+
+=== The Preference Page
+
+To see the preference page for port monitoring, you'll want to go Window -> Preferences -> Run/Debug -> TCP/IP Monitor.  You'll see the page depicted below. 
```

Comment:
you can use `menu:Window[Preferences > Run/Debug > TCP/IP Monitor]` so menu references are consistent across the blogs.